### PR TITLE
2021 TeamD Wiki Entry Submission

### PR DESCRIPTION
**Pull request by Lokera6**

### CODE REVIEW

File: wiki/tools/docker.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
----
-date: 2019-05-16
+---
+date: 2021/04/07
```
Comment (smbryan) [inline wiki/tools/docker.md:2]: Incorrect date format.

File: wiki/tools/docker.md [hunk 2]
```diff
@@ -144,6 +144,100 @@ To generate this message, Docker took the following steps:
...
+```
+Option 2: Remove the container and launch a new one.
+```sh
+sudo docker rm {CONTAINER_NAME}
+sudo docker run -it --name {CONTAINER_NAME} {IMAGE_NAME}:{IMAGE_TAG}
+```
+
+## Other Useful Docker Features
+### Running Multiple Processes
+If you have a container that is running and you want to run another process in that container you can use ```docker exec```. Note that this must be done from the operating system and NOT from within the docker container. For example, to launch another shell you would use
```
Comment (smbryan) [inline wiki/tools/docker.md:190]: Inline code should be `...` and not ```...``` (Jekyll will insert a code block).

**Comment by smbryan** [comment]
This looks like a needed update to a useful article. Just a few format issues to cleanup.

**Comment by Lokera6** [comment]
This last commit should address the formatting issues that you pointed out

**Pull request merged**